maven-settings-builder/pom.xml

```diff
@@ -72,8 +72,12 @@ under the License.
       <plugin>
         <groupId>org.eclipse.sisu</groupId>
         <artifactId>sisu-maven-plugin</artifactId>
+        <!-- <goals>
+          <goal>main-index</goal>
+          <goal>test-index</goal>
+        </goals> -->
```

Comment:
What for is it?

Reply:
Sorry, a mistaken addition (while working on some red squiggly problems)